.werft/jobs/build/installer/installer.ts

```diff
@@ -0,0 +1,237 @@
+import { exec } from "../../../util/shell";
+import { getGlobalWerftInstance } from "../../../util/werft";
+import { getNodePoolIndex } from "../deploy-to-preview-environment";
+
+const blockNewUserConfigPath = './blockNewUsers';
+const workspaceSizeConfigPath = './workspaceSizing';
+const PROJECT_NAME = "gitpod-core-dev";
+const CONTAINER_REGISTRY_URL = `eu.gcr.io/${PROJECT_NAME}/build/`;
+const CONTAINERD_RUNTIME_DIR = "/var/lib/containerd/io.containerd.runtime.v2.task/k8s.io";
+
+export interface Installer {
```

Comment:
In Typescript you generally only use an interface if you intend there to be multiple implementations of the interface. In this case we only have the one so it is preferred to instead just export a class and make your `newInstaller` function the constructor of the class.